depscan/cli.py

```diff
@@ -588,9 +614,22 @@ def main():
     if not args.no_banner:
         print(at_logo)
     src_dir = args.src_dir_image
-    if not src_dir:
+    if not src_dir or src_dir == ".":
         src_dir = os.getcwd()
     reports_dir = args.reports_dir
+    if args.csaf:
+        toml_file_path = os.path.join(src_dir, "csaf.toml")
```

Comment:
So the very first time the user arrives here, we exit with this info. Instead can we leave place holder values and prompt them to edit the document at the end?

Reply:
@prabhu I'd rather not, as that means they will run through the whole scan and then have to do it again. Better to stop the scan at the very beginning so they can do this part and not waste time later. They can always not enter their information and run again if that's what they want to do, as it won't do this if the csaf.toml is present, whether they've filled it out or not.

Reply:
@prabhu I can add a note so that they know they can do that if they are just trying it out.